update changelog (#108)
moov-io/base: CHANGELOG.md

## v0.13.0 (Released 2020-10-16)

> **BREAKING CHANGES**

Reviewer: ```suggestion
**BREAKING CHANGES**

- package `log`: `Logger.LogError` and `Logger.LogErrorf` no longer return an `error`, they will return `LoggedError` which can be called with `Err()` to return an `error`

```

> - database: change interface of New and NewAndMigrate functions (reordered arguments, changed returned parameters)

Reviewer: ```suggestion
- database: changed signature of `New` and `NewAndMigrate` functions by reordering arguments and changing return types
```

- database: renamed Sqlite to SQlite and MySql to MySQL in database config 

ADDITIONS

> - database: load sql files for migrations from /migrations

Reviewer: ```suggestion
- database: load sql files for migrations from `/migrations` directory
```

## v0.13.0 (Released 2020-10-16)

**BREAKING CHANGES**

> - package `log`: `Logger.LogError` and `Logger.LogErrorf` no longer return an `error`, they will return `LoggedError` which can be called with `Err()` to return an `error`

Reviewer: ```suggestion
- log: `Logger.LogError` and `Logger.LogErrorf` no longer return an `error`, they will return `LoggedError` which can be called with `Err()` to return an `error`
```
